src/libsync/bandwidthmanager.cpp

```diff
@@ -85,12 +88,11 @@ BandwidthManager::BandwidthManager(OwncloudPropagator *p) : QObject(),
 
 BandwidthManager::~BandwidthManager()
 {
-    qDebug() << Q_FUNC_INFO;
 }
 
 void BandwidthManager::registerUploadDevice(UploadDevice *p)
 {
-    //qDebug() << Q_FUNC_INFO << p;
+    qCDebug(lcBandwidthManager) << p;
```

Comment:
do we need all these debug?  if they were commented out it probably does not matter